integration_tests/features/environment.py

```diff
@@ -20,10 +20,14 @@ def after_scenario(context, scenario):
 
     if hasattr(context, "exc") and context.exc:
         from traceback import print_exception
+        from sys import _OptExcInfo
```

Comment:
never seen this before

Reply:
Is it coming from runtime?
```py
>>> from sys import _OptExcInfo
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name '_OptExcInfo' from 'sys' (unknown location)
```

Reply:
It's the type of a [`sys.exc_info()`](https://docs.python.org/3/library/sys.html#sys.exc_info) call. I can remove it just to be sure, though.